src/Continuous/normalization.jl

```diff
@@ -507,7 +507,7 @@ end
 _wrap_invariant(X::LazySet, ::Int) = X
 _wrap_invariant(::Nothing, n::Int) = Universe(n)
 
-_wrap_inputs(U::AbstractInput, B::IdentityMultiple) = isidentity(B) ? U : map(u -> B * u, U)
+_wrap_inputs(U::AbstractInput, B::IdentityMultiple) = isidentity(B) ? U : map(u -> scale(B.M.λ, u), U)
```

Comment:
isn't `B * u` not using internally the scale method? what is the advantage of using `scale`?